cilium-cli/connectivity/builder/manifests/template/template.go

```diff
@@ -6,13 +6,22 @@ package template
 import (
 	"bytes"
 	"html/template"
+	"net"
 	"strings"
 )
 
 // Render executes temp template with data and returns the result
 func Render(temp string, data any) (string, error) {
 	fns := template.FuncMap{
 		"trimSuffix": func(in, suffix string) string { return strings.TrimSuffix(in, suffix) },
+		"ipToCIDR": func(ipString string) string {
+			ip := net.ParseIP(ipString)
+			if ip != nil && ip.To4() == nil {
+				return ipString + "/128" // ipv6 address
+			} else {
+				return ipString + "/32" // otherwise assume IPv4
+			}
```

Comment:
Let's use the more modern `net/netip` package here (cf. https://github.com/cilium/cilium/issues/24246):

```suggestion
			ip, err := netip.ParseAdd(ipString)
			if err == nil && ip.Is6() {
				return ipString + "/128"
			} else {
				return ipString + "/32" // otherwise assume IPv4
			}
```